src/logic/Windows.swift

```diff
@@ -243,24 +243,33 @@ class Windows {
         }
     }
 
+    /// Updates windows "lastFocusOrder" to ensure unique values based on window z-order.
```

Comment:
I think that this method was written before the introduction of `lastFocusOrder`, and was not updated when it was added. Thank you for updating it!